src/reconcile/rec_write.c

```diff
@@ -51,6 +51,7 @@ typedef struct {
 
 	/* Lookaside boundary tracking. */
 	uint64_t unstable_txn;
+	wt_timestamp_t unstable_durable_timestamp;
 	wt_timestamp_t unstable_timestamp;
```

Comment:
The `unstable_durable_timestamp` doesn't seem to be being initialized in the same place as the `unstable_timestamp` - was that deliberate?

Reply:
That was not intentional.